packages/components/src/components/Tag/Tag.tsx

```diff
@@ -0,0 +1,8 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+type TagProps = {
+    children: React.ReactNode;
```

Comment:
add `import { type ReactNode } from 'react';`
```suggestion
    children: ReactNode;
```